2017/11/29/systemd-primer-from-the-trenches.html.md

```diff
@@ -0,0 +1,514 @@
+---
+author: Ian Neilsen
+title: "systemd: A primer from the trench's"
```

Comment:
Ian, would you please run this whole article through a spell checker? There are quite a few typos that should be caught by that.

Reply:
YEs

Reply:
done